pysquared/logger.py

```diff
@@ -162,30 +167,15 @@ def _log(self, level: str, level_value: int, message: str, **kwargs) -> None:
 
         json_order.update(kwargs)
 
-        try:
-            json_output = json.dumps(json_order)
-        except TypeError as e:
-            json_output = json.dumps(
-                OrderedDict(
-                    [
-                        ("time", asctime),
-                        ("level", "ERROR"),
-                        ("msg", f"Failed to serialize log message: {e}"),
-                    ]
-                ),
-            )
```

Comment:
This became no longer necessary after #228 but was left in. It has no tests and is no longer required so I removed it.